src/main/java/me/botsko/prism/database/mysql/MySqlPrismDataSource.java

```diff
@@ -106,6 +109,9 @@ private static void setupDefaultProperties(@Nonnull ConfigurationSection section
                 }
             }
             try {
+                if (!propFile.getParentFile().exists() && !propFile.getParentFile().mkdirs()) {
+                        Prism.log("Prism Directory couldn't be created");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['if' child has incorrect indentation level 24, expected level should be 20.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5472183)